addons/web/static/src/views/fields/many2one/many2one_field.js

```diff
@@ -317,8 +329,8 @@ Many2OneField.extractProps = ({ attrs, field }) => {
         canOpen: !noOpen,
         canCreate,
         canWrite,
-        canQuickCreate: canCreate && !noQuickCreate,
-        canCreateEdit: canCreate && !noCreateEdit,
+        canQuickCreate: Boolean(canCreate && !noQuickCreate),
+        canCreateEdit: Boolean(canCreate && !noCreateEdit),
```

Comment:
Same not need boolean

Reply:
We need those props to be false instead of undefined because otherwise they will fallback to the default value which is true

Reply:
Francois is right, both values are already booleans, so no need to cast their conjunction

Reply:
Mmh, I didn't know that this is how it worked, my bad